sdks/python/apache_beam/runners/portability/stager.py

```diff
@@ -200,10 +201,11 @@ def stage_job_resources(self,
     # Handle jar packages that should be staged for Java SDK Harness.
     jar_packages = options.view_as(
         DebugOptions).lookup_experiment('jar_packages')
+    classpath_separator = ':' if platform.system() != 'Windows' else ';'
```

Comment:
Can we use the same class path separator (';') for all platforms?

Reply:
I think there’s no issue to use ‘;’ on all platforms. ‘:’ is a standard classpath separator on Linux systems so some people on Linux may find that using ‘;’ for separating jar files makes them feel awkward but it’s not a classpath anyway. Do you think it would be better to use a single character of choice for all platforms?

Reply:
I think it will be simpler but I am not a java developer either. At the same time, this feature will likely be used by python developers for cross language using java transforms.

Feel free to check with another person who would have a better idea.

Reply:
@chamikaramj WDYT?

Reply:
Yeah, let's use a single separator (prefer colon) to make things simpler and offer a platform-independent API.

Reply:
We can't use colon. It won't work on Windows(colon is a drive name suffix). We can use semicolon though.

Reply:
Class path separators are system dependent, for windows it will be `;`
Reference: https://stackoverflow.com/questions/4528438/classpath-does-not-work-under-linux

Reply:
+1 for ','. Thanks for the suggestion.

Reply:
+1